aten/src/ATen/native/cuda/PointwiseOpsKernel.cu

```diff
@@ -25,6 +31,11 @@ void addcmul_cuda_kernel(TensorIteratorBase& iter, const Scalar& value) {
         auto alpha = value.to<scalar_t>();
         static const auto addcmul_string = jiterator_stringify(
           template <typename T> T addcmul(T a, T b, T c, T alpha) { return a + alpha * (b * c); });
+        if (iter.is_cpu_scalar(3)) {
```

Comment:
The code duplication is making me sad but I don't see a trivial way to detangle the AT_DISPATCH....

oh well maybe we don't worry about that this PR (as there was similar code duplication in lerp too)

Reply:
Do you think a function that takes in iter and the position of the cpu scalar and calls the scalar kernel would be better to reduce duplication?

Reply:
Oh I just realized why that wouldn't work since the return wouldn't properly exit the function without some boilerplate

Reply:
Yea haha it's the scalar_t that the dispatch provides that we no longer have.